src/native/corehost/apphost/common_controls.manifest

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
```

Comment:
Would it also be possible to set the default DPI awareness in the manifest, like this:

```suggestion
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
  <asmv3:application>
    <asmv3:windowsSettings>
      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/pm</dpiAware>
      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2,PerMonitor</dpiAwareness>
    </asmv3:windowsSettings>
  </asmv3:application>
  
```

It will prevent the dialog box from being blurry on high DPI monitors (like it is in the image in the PR description). There aren't any code changes required since MessageBox (and Task Dialog) knows how to scale itself when the proper DPI mode is selected. (The specific set of options here makes it compatible with all of the legacy high DPI modes in Windows 7 and 8.x.)

Reply:
Unfortunately, that seems to be one of those settings that has to be in the process manifest and doesn't work with programmatically creating/activating a context.

It might be worth looking into what we can do and what it would cost though. I did see the APIs available starting with one of the Win10 updates that allow controlling DPI awareness for the thread, but I'd rather keep investigating that separate from getting a manifest / activation context in for modern theming.

Reply:
Oh—that's too bad. I see now from the docs that the activation context only holds the SxS information. I am sorry, I should have tried with the activation context before suggesting this.

Reply:
I think it's a great suggestion, and it's unfortunate it doesn't work the simple way. I agree that we should investigate the APIs a separate work item - if nothing else it would need some light-up code and so on.